wordclock_tools/wordclock_display.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, config, wci):
         if config.getboolean('wordclock', 'developer_mode'):
             import wordclock_tools.wordclock_strip_gtk as wcs_gtk
             self.strip = wcs_gtk.GTKstrip(wci)
+        elif config.getboolean('wordclock', 'developer_mode_macos'):
```

Comment:
If WXstrip works platform independent, I'm happy to replace GTKstrip. ...to avoid multiple tools/modes doing actually the same thing..

Reply:
resolved